src/assets/githubAction/githubActionBeginner.md

```diff
@@ -0,0 +1,129 @@
+
+### 2. Events:
+Events are specific activities such as code push, pull request creation, issue creation, comments on pull requests or issues, etc., that trigger a workflow.
+
+### 3. Jobs
+Jobs are sets of steps that are executed in the same runner. A workflow with multiple jobs will run in parallel, which is default, but it can be configured so that jobs run sequentially.
+
+### 4. Steps
+A step is an individual task that can run the commands in a job. These can be actions or commands.
+
+### 5. Actions
+Actions are the standalone commands for the GitHub Actions platform that perform complex but frequently repeated tasks. Actions help to reduce repetitive code in workflows.
+
+### 6. Runner
+A runner is a GitHub Actions server where our workflows run. It listens for available jobs, runs multiple jobs at a time, and reports the progress, logs, and results back to GitHub. GitHub hosted runners are based on Ubuntu, Linux, Microsoft Windows, and Mac OS.
+
+Note:
+One event can trigger many workflows, a workflow can contain many jobs, and a job can contain many steps.
+
+### Process in GitHub Actions:
+
+![process_in_githubActions](/src/assets/githubAction/images/process.png)
+
+## Getting started:
+
+Before we dive into an example, make sure you have a repository where you want to set up your CI/CD workflow. If you don't have one, create a new repository, for example, named `action-hero`.
+
+Step 1: Clone your repository into your system
+```bash
+git clone <your_github_repo_url>
+```
+
+Step 2: Go to path `action-hero` and create a folder named `.github`
+```bash
+cd action-hero
+mkdir .github
+```
+
+Step 3: Then, go to path `.github` and create a folder named `workflows`
+```bash
+cd .github
+mkdir workflows
+```
+
+Step 4: Inside the folder `.github`, create a subfolder named `workflows`
+```bash
+cd workflows
+touch hello_world.yml
+```
+Our file structure should look like this
+```
+📦Action-hero
+┗ 📂.github
+┃ ┣ 📂workflows
+┃ ┃ ┗ 📜hello_world.yml
+```
+
+Step 5: Add the following code snippet to your file `hello_world.yml`:
+
+```
+name: Hello world workflow
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Print a message
+      run: echo "Hello, GitHub Actions!"
+```
+
+In this example:
+- The workflow is triggered by a push event on the main branch.
+- The job named "build" runs on the latest version of Ubuntu.
+- The first step checks out the repository using the `actions/checkout` action.
+- The second step simply prints a message using the `echo` command.
+
+## GitHub Actions activity:
+
+![github activity](/src/assets/githubAction/images/output.png)
+
+The image above is an output from a GitHub Actions workflow run. It shows that the workflow was successful and completed the `Hello world workflow` job, which consisted of two steps: `Checkout repository` and `Print a message`.
```

Comment:
```suggestion
The image above is the output from our GitHub Actions workflow run. It shows that the workflow was successful and completed the `Hello world workflow` job, which consisted of two steps: `Checkout repository` and `Print a message`.
```

Since you're referring to a specific workflow, not just any one, it's better to use a specific article (the) instead of an unspecific article (a).